test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "proposition": 0
+}
```

Comment:
I don't think you meant to include these test.(json/lcgs) files

Reply:
Right. Fixed in 827281b